Client.xcodeproj/xcshareddata/xcschemes/Fennec.xcscheme

```diff
@@ -147,7 +147,7 @@
       </PreActions>
       <Testables>
          <TestableReference
-            skipped = "NO">
+            skipped = "YES">
```

Comment:
So, with this we skip the XCUITests when building on BB, but we still run the unit tests...would that be a valid approach? it takes less than 30 minutes to run those...And going to Fennec schema -> Edit -> Test -> mark XCUITests, the enabled XCUITests are available to run...

Reply:
Sounds good. Modifying the scheme to run XCUITests is an extra step I was hoping to avoid, but we'll have to go with this approach.